<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageNetwork</class>
 <widget class="QWidget" name="PageNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>379</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>608</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>VTX</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="modeLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="0" rowspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Channel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" rowspan="2" colspan="3">
          <widget class="QLabel" name="vtxFrequency">
           <property name="text">
            <string>-1 MHz</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="3">
          <widget class="QLabel" name="vtxPowerDbm">
           <property name="text">
            <string>-1 dBm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <widget class="QPushButton" name="vtxChannelDecr">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" rowspan="2">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Frequency</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="2" rowspan="2">
          <widget class="QSpinBox" name="vtxPowerSpin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="8" column="1" rowspan="4" colspan="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Power</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3" rowspan="2">
          <widget class="QPushButton" name="vtxPowerIncr">
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" rowspan="3">
          <widget class="QPushButton" name="vtxReset">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" rowspan="2">
          <widget class="QPushButton" name="vtxPowerDecr">
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" rowspan="2">
          <widget class="QSpinBox" name="vtxChannelSpin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="vtxChannelIncr">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="11" column="4">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" rowspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Power</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="4" rowspan="3">
          <widget class="QPushButton" name="vtxSet">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="8">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>vtxChannelDecr</sender>
   <signal>clicked()</signal>
   <receiver>vtxChannelSpin</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vtxChannelIncr</sender>
   <signal>clicked()</signal>
   <receiver>vtxChannelSpin</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vtxPowerIncr</sender>
   <signal>clicked()</signal>
   <receiver>vtxPowerSpin</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vtxPowerDecr</sender>
   <signal>clicked()</signal>
   <receiver>vtxPowerSpin</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
